src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/pensjon/HistoriskPensjonService.kt

```diff
@@ -1,21 +1,25 @@
 package no.nav.familie.ef.sak.opplysninger.personopplysninger.pensjon
 
 import no.nav.familie.ef.sak.fagsak.FagsakPersonService
-import no.nav.familie.ef.sak.fagsak.FagsakService
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.PersonService
 import org.springframework.cache.annotation.CacheConfig
 import org.springframework.cache.annotation.Cacheable
 import org.springframework.stereotype.Service
 import java.util.UUID
 
 @Service
 @CacheConfig(cacheManager = "longCache")
-class HistoriskPensjonService(private val historiskPensjonClient: HistoriskPensjonClient, val fagsakPersonService: FagsakPersonService) {
+class HistoriskPensjonService(
+    private val historiskPensjonClient: HistoriskPensjonClient,
+    val fagsakPersonService: FagsakPersonService,
+    val personService: PersonService
+) {
 
     @Cacheable("historisk_pensjon")
     fun hentHistoriskPensjon(fagsakPersonId: UUID): HistoriskPensjonResponse {
         val aktivIdent = fagsakPersonService.hentAktivIdent(fagsakPersonId)
-        val identer = fagsakPersonService.hentIdenter(fagsakPersonId)
-        return historiskPensjonClient.harPensjon(aktivIdent, identer.map { ident -> ident.ident })
+        val identer = personService.hentPersonIdenter(aktivIdent)
+        return historiskPensjonClient.harPensjon(aktivIdent, identer.identer.map { pdlIdent -> pdlIdent.ident })
```

Comment:
Det finnes en støttemetode for å trekke ut identene fra `hentPersonIdenter` :) 
```suggestion
        val identer = personService.hentPersonIdenter(aktivIdent).identer()
        return historiskPensjonClient.harPensjon(aktivIdent, identer)
```